examples/advanced-ops-outbrain/.ipynb_checkpoints/01-Download-Convert-checkpoint.ipynb

```diff
@@ -0,0 +1,417 @@
+{
```

Comment:
We shouldn't check in the checkpoints - can you remove all the '.ipynb_checkpoints' files?

Reply:
I have deleted the checkpoints and updated the .gitignore so that they are not included anymore when is subfolders